src/index.js

```diff
@@ -1,3 +1,2 @@
-export * from "./common/index.js";
 export * from "./cards/index.js";
-export { getStyles, getAnimations } from "./getStyles.js";
+export * from "./generators/index.js";
```

Comment:
Any reason why you're exporting these? These should be internal functions, and not exported. Keeping these as is bloats up the NPM package when I use `import *`, since everything from the `CONSTANTS` to the `utils` functions gets loaded in.

Reply:
I wanted to allow other developers to use these functions when they implement new cards. You are right however if the npm package is setup the right way people can just use the `Card` class. 👍🏻